src/initialize/checkpoint.sh

```
@@ -0,0 +1,62 @@
+		_checkpoint_time=$time
+
+	elif [[ -n "$UXMODE" && "$type" == mark ]]; then
+		_checkpoint_time=$(date +%s)
+		echo "$msg"
+	fi
+}
+
+_checkpoint_help()
+{
+	echo "
+Usage: ${module_options[checkpoint,feature]} <action> <message>
```

Shellcheck (0.10.0)

[warning] module_options is referenced but not assigned.

(SC2154)